system/Filters/Filters.php

```diff
@@ -60,6 +60,11 @@ class Filters
 		'after'  => [],
 	];
 
+	protected $filtersClass = [
+		'before' => [],
+		'after'  => [],
+	];
+
```

Comment:
new property can be added:

```suggestion

           /** @var array */
          protected $argumentsClass;

```